android/orblibrary/src/main/java/org/orbtv/orblibrary/IOrbSessionCallback.java

```diff
@@ -898,54 +835,7 @@ default void onReceiveError(int code, String message,
         throw new UnsupportedOperationException("Unsupported 204 API.");
     }
 
-    /**
-     * @since 204
-     *
-     * Request for the description of the current media on applications
-     *
-     * @return true if this event has been handled, and false if not
-     */
-    default boolean onRequestMediaDescription() {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
-    }
-
-    /**
-     * @since 204
-     *
-     * Request to deliver a text input, from voice command, to applications
-     *
-     * @param input The content of the text
-     * @return true if this event has been handled, and false if not
-     */
-    default boolean onRequestTextInput(String input) {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
-    }
-
-    /**
-     * Called to send an intent, from a voice command, to applications
-     *
-     * @param action The index number of the intent, from intent.media.pause to intent.playback
-     * @param info   The value uniquely identifying a piece of content:
-     *               - INTENT_MEDIA_SEEK_WALLCLOCK: a wall clock time
-     *               - INTENT_DISPLAY: a URI
-     *               - INTENT_SEARCH: a search term specified by the user.
-     *               - INTENT_PLAYBACK: a URI
-     * @param anchor The value indicates an anchor point of the content, which is either "start" or "end"
-     * @param offset The number value for the time position, a number of seconds
-     * @return true if this event has been handled, and false if not
-     */
-    default boolean onSendIntentByVoiceCommand(Integer action, String info, String anchor, int offset) {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
-    }
-
-    /**
-     * @since 204
-     *
-     * Called to send a send a key press event, from a voice command, to the application 
-     *
-     * @param action The index number of the intent, either pressing a button or showing a log
-     */
-    default boolean onSendKeyPressAction(Integer action) {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
+    default void consoleLog(String log) {
+        Log.d("", "consoleLog: " + log);
```

Comment:
need a comment

Reply:
Done, thank you.

Reply:
there are two ways
One makes it a specific method for 204
```java
    /**
     * @since 204
     *
    ....
     */
    default void consoleLog(String log) {
        throw new UnsupportedOperationException("Unsupported 204 API.");
``` 

or just make it a general method (better to give it a TAG, like "IOrbSessionCallback")
```java
    /**
     *
    ....
     */
    default void consoleLog(String log) {
        Log.d("", "consoleLog: " + log);
    }
```

Reply:
I prefer the 1st one. How do you think?

Reply:
I've just confirmed with Josh, we think the second approach is better. I'll update the TAG to "IOrbSessionCallback".

Reply:
Got it. Thanks.